chicken-sloths/bangazon-site-sprint3
# delete from cart

# Description
You can delete products from your active order/shopping cart.

## Related Ticket(s)
fixes #20, fixes #29

## Steps to Test Solution
1. `npm run db:gen`
1. `npm start`
1. Log in as anybody (`Linnea_Batz75@hotmail.com`)
1. View cart.
1. Delete all the shit.
1. Refresh the page to make sure that shit is gone.
